index.html

```diff
@@ -370,9 +370,15 @@ <h3>Ecosystem Overview</h3>
           <dd>
 A role an [=entity=] performs by asserting [=claims=] about one or
 more [=subjects=], creating a [=verifiable credential=] from these
-[=claims=], and transmitting the [=verifiable credential=] to a
-[=holder=]. Example issuers include corporations, non-profit organizations,
+[=claims=], and transmitting the [=verifiable credential=] to the
+[=issuee=]. Example issuers include corporations, non-profit organizations,
```

Comment:
Hmm, it could be that an issuer hands the credential to some party *other than the issuee* as the first holder. And that party will / is expected to perform delivery to the `issuee`. At least, I would expect people to think that the `issuee` is different from this intermediary. I worry that this change reduces something more general to a specific case where the issuer interacts directly with the "issuee" whereas the previous text allowed for there to be any number of intermediaries.

If an issuer issues into a repository managed by a third party that holds the credentials for later pickup by eventual recipients -- is that third party the issuee ... or are those recipients the issuees?

Issuers may also issue directly onto public lists, where I don't think the "public list" would be considered an "issuee", but it would be a "holder". There may be no "issuee" in this case ... right?

I think we would need different language here around the "issuee" case being a more specific case (even if it happens quite often) of the more general model. So the issuer always issues to a holder, which may / may not be expressed or recognized as an issuee?

Reply:
@dlongley — Good questions! Definitely need to be addressed!

(Such questions are part of why I suggested there be an issue [per repo] for this, as comments on an issue will be more visible in the future than inline PR review comments.)

Reply:
After submitting this PR, I (privately) raised with Manu the issue of making changes to our multiple documents now  and he recommended waiting for this PR to be resolved before making any changes to any other of our documents